### PR TITLE
Update plugin Phone Spain

### PR DESCRIPTION
**Pull request by JorgeSanzS**
Update the phone plugin to be compatible with Spain.

### CODE REVIEW

**Comment by jocelynj** [comment]
@JorgeSanzS: We just added some automatic tests on osmose-backend, and it seems that your new tests are failing. Do you need help to fix them or run tests?

File: plugins/Phone.py [hunk 1]
```diff
@@ -252,3 +252,29 @@ class father:
         # Verify we got error for other correct numbers
         for bad in (u"3631", u"(123) 123-4567", "+1 123 1234567"):
             assert p.node(None, {"phone": bad}), ("phone='%s'" % bad)
+    def test_ES(self):
```
Comment (frodrigo) [inline plugins/Phone.py]: Plase, skip a line before def.

**Comment by frodrigo** [comment]
Tu run the test, just run
```sh
nosetests plugins/Phone.py
```

File: plugins/Phone.py [hunk 2]
```diff
@@ -252,3 +252,29 @@ class father:
...
+        p = Phone(None)
+        class _config:
+            options = {"country": "ES", "phone_code": "34", "phone_len": 9, "phone_format": r"^[+]%s[- ][0-9]{3}[- ][0-9]{3}[- ][0-9]{3}$"}
```
Comment (frodrigo) [inline plugins/Phone.py]: Here is only the test settings. The real setting are in `osmose_config.py`

**Comment by frodrigo** [comment]
Cherry-picked and fixed in 88055ef20359e456617ac0a9c6d416b70c39dfca

**Comment by frodrigo** [comment]
It's done. The results are available.

Please report here any false positive.

I have disable the check of the number format. After reading Wikipedia, there is many possible formats.

**Comment by frodrigo** [comment]
Spain is now the biggest country not ussing international prefix
http://osmose.openstreetmap.fr/en/errors/?item=3092

**Pull request closed without merging**